bourbonandburial/BourbonAndBurial
# Ml profile updates

closes #63 
How to test

1. log in with an active user
2. go to user profile
3. click the trash can icon  (user should be logged out and taken  to sign in page)
4. check database to see if isActive is set to 0
5. log into app with the same deactivated user
6. user is prompted with modal to fill out information
7. user is updated in database but not duplicated
